src/components/RegistrationFlow/Description/index.js

```diff
@@ -7,13 +7,13 @@ import './styles.scss'
 const MAX_LENGTH = 200
 
 const Description = props => {
-  const { onChange, value } = props
+  const { onChange, value, hideStep } = props
   return (
     <ShadowBox>
       <main role="main" className="add-user-description-container">
         <div className="profile-creation-title-container">
           <h2 className="profile-creation-title">Kerro itsestäsi</h2>
-          <span className="profile-creation-step-text">4/6</span>
+          {!hideStep && <span className="profile-creation-step-text">4/6</span>}
```

Comment:
This is to hide the step? You probably forgot to do this in the interests-section, it still shows 6/6.

Reply:
Yes! Added hideStep also for Interests editing.

Reply:
Also fixed situation where user might not yet have any direct channels! Well noted!